src/GraphQL.php

```diff
@@ -211,14 +211,15 @@ public static function getStandardTypes(): array
      * Standard types not listed here remain untouched.
      *
      * @param array<string, ScalarType> $types
+     * @param bool $partial Whether to replace only types listed in $types or all standard types
```

Comment:
This makes little sense to me. If this method were called with `types: ['ID' => $newIDType], partial: false`, what would that even mean? How could a subset of the required standard types fully replace all standard types?

Reply:
Imagine you have 2 schema's.

In schema 1 you override ID type.

In schema 2 you override String type.

Schema 1 doesn't want to get schema's 2 string type at any point.
Schema 2 doesn't want to get schema's 1 ID type at any point.

With normal PHP FPM this is not an issue as only one schema is active within a request and destroyed afterwards.
But when doing unit tests, or even worse, functional tests, you it could be that one test used schema 1, that initialized the standard types. Then later a test uses schema 2, and boom. It breaks.

So the only way to be able to do this, is to wipe the cache.

Or, to explicitly reinitialize the types that are not overridden, so that the array always contains the full standard types.

Reply:
I guess I can also solve it like this:
```php
Type::overrideStandardTypes([
            Type::INT => isset($types[Type::INT]) ? $loader(Type::INT) : new IntType(),
            Type::FLOAT => isset($types[Type::FLOAT]) ? $loader(Type::FLOAT) : new FloatType(),
            Type::STRING => isset($types[Type::STRING]) ? $loader(Type::STRING) : new StringType(),
            Type::BOOLEAN => isset($types[Type::BOOLEAN]) ? $loader(Type::BOOLEAN) : new BooleanType(),
            Type::ID => isset($types[Type::ID]) ? $loader(Type::ID) : new IDType(),
        ]);
```

Reply:
@spawnia Just confirmed, that this also works. So I removed the partial thing from the PR.

It's ready for review again!